tests/translate/translate_vorticityvolumemean.py

```diff
@@ -1,18 +1,25 @@
-import fv3core.stencils.vorticity_volumemean as vm
+from fv3core.stencils.d_sw import vorticity_from_winds
 from fv3core.testing import TranslateFortranData2Py
 
 
 class TranslateVorticityVolumeMean(TranslateFortranData2Py):
     def __init__(self, grid):
         super().__init__(grid)
-        self.in_vars["data_vars"] = {"u": {}, "v": {}, "ut": {}, "vt": {}, "wk": {}}
+        self.in_vars["data_vars"] = {"u": {}, "v": {}, "wk": {}}
         self.out_vars = {
             "wk": {},
-            "ut": grid.x3d_domain_dict(),
-            "vt": grid.y3d_domain_dict(),
         }
 
     def compute(self, inputs):
         self.make_storage_data_input_vars(inputs)
-        vm.compute(**inputs)
+        vorticity_from_winds(
+            inputs["u"],
+            inputs["v"],
+            self.grid.dx,
+            self.grid.dy,
+            self.grid.rarea,
+            inputs["wk"],
+            origin=(0, 0, 0),
```

Comment:
self.grid.default_origin() is preferred for consistency

Reply:
Updated.

We should maybe consider renaming this later, it was unclear to me from the name whether this meant start of halo data or start of compute domain. And looking at the code didn't really help, because I wasn't sure what all the single-letter-append after is/ie meant.

Reply:
'default' or 'standard' refer to the domain isd:ied, jsd:jed,0:npz-1 inclusive, while names with 'compute' in them refer to the is/ie, js/je compute region. By single letter append, do you mean the d in isd? That is just to follow the conventions of the fortran code naming of domain indices.  Renaming all of it is of course open to discussion to do at some point. As we merge stencils they will become more compute region centric, in which case using 'default' or 'standard' to name the full domain including halos will make less sense.